mapswipe-dashboard/app/components/CalendarHeatMapContainer/index.tsx

```diff
@@ -46,18 +47,21 @@ interface Data {
 
 interface Props {
     data: Data[] | undefined | null;
-    maxContribution?: number;
 }
 
 function CalendarHeatMapContainer(props: Props) {
     const {
         data,
-        maxContribution = 1000,
     } = props;
     const range = getDateRange(data);
 
+    useEffect(() => {
+        ReactTooltip.rebuild();
+    });
+
+    const maxContributionValue = Math.max(5, ...(data?.map((d) => d.count) ?? []));
```

Comment:
Any reason we have `5` here?

Reply:
we need to bin contribution to at least 5 different bins for five different color values to make sense.

Reply:
Let's add a comment